index.js

```diff
@@ -92,11 +92,12 @@ const staticify = (root, options) => {
 
         const fileName = path.basename(p);
         const fileNameParts = fileName.split('.');
+        const fileNameHash = fileNameParts[fileNameParts.length - 2];
         const re = new RegExp(`^[0-9a-f]{${HASH_LEN}}$`, 'i');
+        const reResult = re.exec(fileNameHash);
 
-        if (fileNameParts.length >= 3 &&
-            fileNameParts[fileNameParts.length - 2].length === HASH_LEN &&
-            re.exec(fileNameParts[fileNameParts.length - 2])[0] === fileNameParts[fileNameParts.length - 2]
+        if (fileNameParts.length >= 3 && fileNameHash.length === HASH_LEN &&
+            (reResult && reResult[0] === fileNameHash)
         ) {
             const stripped = fileNameParts.slice(0, fileNameParts.length - 2);
```

Comment:
Maybe use `fileNameHash` here too?

Reply:
This line returns the part before the hash, e.g `script`, whereas `fileNameHash` will return the hash itself, e.g. `4e2502b` so that wouldn't work here